decompiler/config/jak2/all-types.gc

```diff
@@ -6953,6 +6953,8 @@
   (text-x24d #x024d)
   (text-x24e #x024e)
   (text-x24f #x024f)
+  (text-781 781)
+  (text-782 782)
```

Comment:
can you use hex for these instead?

Reply:
I'd rather not, since all of our enums use decimal, it creates very awkward merge conflicts for people to follow the convention used everywhere else and then have to reorder decimal/hex when merging.

I will give these proper names though.

Reply:
All the other entries in the `text-id` enum are hexadecimal though, and hexadecimal is also what's used in the text tools.